packages/application-generic/src/services/in-memory-provider/providers/variable-mappers.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Due some problems with Azure Redis DB that doesn't allow for certain
+ * configuration values to be empty or have an empty string and as we don't
+ * want to process them in our provider configuration files, we implement
+ * this mapper function to be able to overcome that limitation in Azure
+ * temporarily while we find a better solution
+ */
+export const convertStringValues = (
```

Comment:
Why do you have this function written twice?

Reply:
I forgot to remove it from the initial location in the index file. Thanks for spotting it.